tests/shared/src/test/scala/cats/tests/SyntaxSuite.scala

```diff
@@ -269,6 +269,28 @@ object SyntaxSuite {
     tfa.parFlatMap(mfone)
   }
 
+  def testApplyN[F[_]: Apply, A, B, C, T] = {
+    val fa = mock[F[A]]
+    val fb = mock[F[B]]
+    val fc = mock[F[C]]
+
+    val fapply2 = mock[(A, B) => T]
+
+    val result2 = fapply2.applyN(fa, fb)
+
+    result2: F[T]
```

Comment:
FWIW: I like Arman's suggestion about naming. It really seems like `liftN`.

Reply:
yeah, `liftN` makes sense to me

Reply:
My 🪙🪙: in the `parFlatMapN` PR I did @johnynek pointed out that in these syntax tests there's always the 1-arity case lacking, it would maybe make sense to add it. [ref](https://github.com/typelevel/cats/commit/fe96ffaad1f287a88712e5bee95c58d0a4a68720)

Reply:
Yeah, I think I'll have a test for each arity case recognized by the codegen separately

Reply:
Added a test for arity 1 of each of those, and renamed `applyN` to `liftN`.

Let's get back to bikeshedding :D

Reply:
hoping we can avoid the bikeshed after all. Suggestion applied, closing thread